packages/query/src/bitcoin/ordinals/brc20/brc20-tokens.query.ts

```diff
@@ -59,18 +59,21 @@ export function useGetBrc20TokensQuery({
           })
         );
 
-        // Initialize token with token data
         return brc20Tokens.data.map((token, index) => {
           return {
-            balance: null,
+            balance: token,
             holderAddress: address,
-            marketData: null,
-            tokenData: { ...token, ...tickerPromises[index].data },
+            info: tickerPromises[index].data,
           };
         });
       });
 
-      const brc20Tokens: Brc20Token[][] = await Promise.all(brc20TokensPromises);
+      const brc20Tokens = await Promise.all(brc20TokensPromises).catch(error => {
+        // TODO: add analytics here
```

Comment:
Are we able to add analytics in here? Or can we come up with a solution for this. 

Maybe even just opening an issue to migrate them? I saw this in another PR too. 

Its fine to defer doing this for a while but it could start piling up and also we need analytics if we are to use these packages and take the code out of the extension.

Reply:
I guess this task covers it ? https://github.com/leather-wallet/issues/issues/43